extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -406,13 +427,22 @@ export class AmpLightboxGallery extends AMP.BaseElement {
   toggleDescriptionOverflow_() {
     const measureOverflowState = state => {
       state.isStandard = this.descriptionBox_.classList.contains('standard');
+      state.isOverflown = this.descriptionBox_.classList.contains('overflow');
+
+      // The height of the description without overflow is set to 4 rem.
+      // The height of the overflow mask is set to 1 rem. We allow 3 lines
+      // for the description and consider it to have overflow if more than 3
+      // lines of text.
+      state.hasOverflow = this.descriptionBox_./*OK*/scrollHeight
```

Comment:
What's the difference between the meaning of `isOverflown` and `hasOverflow`?

Reply:
* `hasOverflow` means that the description is longer than the default number of lines we show (we show 4 lines). 
* `isOverflown` means that the description is currently in overflow state, meaning that we apply a shim and enable scrolling. 
Let me think if there's a better way to write this logic.

Reply:
Suggested rename:
`hasOverflow` -> `descriptionOverflows`